content/en/docs/reference/kubectl/conventions.md

```diff
@@ -77,6 +77,6 @@ flag, which provides the object to be submitted to the cluster.
 
 ### `kubectl apply`
 
-* You can use `kubectl apply` to create or update resources. However, to update a resource you should have created the resource by using `kubectl apply` or `kubectl create --save-config`. For more information about using kubectl apply to update resources, see [Managing Resources](/docs/concepts/cluster-administration/manage-deployment/#kubectl-apply).
+* You can use `kubectl apply` to create or update resources. However, to update a resource you should have created the resource by using `kubectl apply`. For more information about using kubectl apply to update resources, see [Managing Resources](/docs/concepts/cluster-administration/manage-deployment/#kubectl-apply).
```

Comment:
@Liujingfang1, is the second sentence required, `However, to update a resources you should`...

Reply:
It can be removed. Will update it.

Reply:
updated